File: 0283-move-zeroes/0283-move-zeroes.c
void moveZeroes(int* nums, int numsSize) {
    short c=0;
    for(int i=0;i<numsSize;i++){
       if(nums[i]!=0){
        nums[c]=nums[i];
        c++;
       }
    }
    for(int i=c;i<numsSize;i++){
        nums[i]=0;
    }
    
}

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
      int[] arr = Arrays.copyOf(nums,nums.length);
      Arrays.fill(nums, 0);
      int index=0;
      for(int itr=0;itr<nums.length;itr++){
        if(arr[itr]!=0)
        nums[index++]=arr[itr];
      }  
    }
}